fractal/cmd/_workflow.py

```diff
@@ -207,10 +207,16 @@ async def patch_workflow(
     *,
     project_id: int,
     workflow_id: int,
-    **workflow_update_dict,
+    new_name: Optional[str] = None,
```

Comment:
Note that the command reads
```
fractal workflow edit [-h] [--name NAME] project_id workflow_id
```
Thus I think we should simply make `new_name` required. Otherwise there's no point in calling this function.